src/other/utils.jl

```diff
@@ -96,95 +95,54 @@ end
 #'
 #' DataFrames.gennames(10)
 function gennames(n::Integer)
-    res = Vector{Symbol}(n)
+    res = Array{Symbol}(n)
     for i in 1:n
         res[i] = Symbol(@sprintf "x%d" i)
     end
     return res
 end
 
-#' @description
-#'
-#' Count the number of missing values in an Array.
-#'
-#' NOTE: This function always returns 0.
-#'
-#' @field a::Array The Array whose missing values are to be counted.
-#'
-#' @returns count::Int The number of missing values in `a`.
-#'
-#' @examples
-#'
-#' DataFrames.countna([1, 2, 3])
-countna(a::Array) = 0
 
 #' @description
 #'
-#' Count the number of missing values in a DataArray.
+#' Count the number of null values in an array.
 #'
-#' @field da::DataArray The DataArray whose missing values are to be counted.
+#' @field a::AbstractArray The array whose missing values are to be counted.
 #'
-#' @returns count::Int The number of missing values in `a`.
+#' @returns count::Int The number of null values in `a`.
 #'
 #' @examples
 #'
-#' DataFrames.countna(@data([1, 2, 3]))
-countna(da::DataArray) = sum(da.na)
+#' DataFrames.countnull([1, 2, 3])
+function countnull(a::AbstractArray)
```

Comment:
Seems like this should either be defined in Nulls or deprecated in favor of `count(isnull, a)`

Reply:
I think it's `countnull` to allow, e.g. `DataArray`/`NullableArray`/`CategoricalArray` to define optimized versions.

Reply:
You still can quite easily:
```julia
Base.count(::typeof(isnull), a::SweetArray) = # awesome implementation
```

Reply:
Yeah, let's remove it in a subsequent PR.